### PR TITLE
Dakota

### PR DESCRIPTION
**Pull request by dakotaihli**
Cleaned up data collection scripts, separated data collection and data merging/extraction into separate files. Data is saved to data/ in RDS format for convenience, and does not retrieve the data if it detects the RDS file. This is so DataCollection.R only needs to be run once per user, since traits::betydb_query takes forever to run.

### CODE REVIEW

**Review by swester3 — APPROVED** [review]

File: src/DataMerging.R [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+#Use managements_treatments to join planting data to yield data
+mt <- readRDS('data/managements_treatments.rds')
+yields <- readRDS('data/yields.rds')
+yieldVersusDensity <- left_join(yields,left_join(planting,mt,by='management_id'),by='treatment_id')
+saveRDS(yieldVersusDensity, file='data/yieldVersusDensity.rds')
+
+#Join species,sites data to certain subfields of yields
+#yieldsSpecSites <- select(yields, id, date, mean, n, statname, stat, site_id, specie_id, treatment_id, citation_id, cultivar_id) %>% 
+#  left_join(species, by = 'specie_id') %>%
```
Comment (klajbor2) [inline src/DataMerging.R:26]: Why was this commented out?

Comment (dakotaihli) [reply src/DataMerging.R:26]: It was suggested that we add it but it's not immediately apparent why we need it (and it overcomplicates our dataframes). If we do need it after all then we can just uncomment it.

Comment (klajbor2) [reply src/DataMerging.R:26]: sounds good!

maybe add a comment on what would trigger uncommenting it?

Comment (klajbor2) [reply src/DataMerging.R:26]: or why it's commented out at the moment

**Review by wlinz — APPROVED** [review]

**Pull request merged**